### ISSUE TITLE
Use Virtual Scroll in Picker Results

### ISSUE THREAD

**Issue by amrutha-rajiv** [bullhorn/novo-elements#844]
_**Note: for support questions, please use one of these channels:**_ [Chat: BullhornOSS Gitter](https://gitter.im/bullhorn/Open-Source)

## Steps to Reproduce the Problem

  1.
  1.
  1.

## Specifications

  - Version:
  - Platform:
  - Subsystem:

**Comment by bvkimball** [comment]
I think the right solution might be to let the picker results template add it.

**Comment by amrutha-rajiv** [comment]
I agree. They're all here https://github.com/bullhorn/novo-elements/tree/master/projects/novo-elements/src/elements/picker/extras
